src/components/stories/AnimatedCover.tsx

```
@@ -33,21 +35,28 @@
-    <div className="h-full w-full bg-cover bg-no-repeat perspective-[1000px]">
-      <div className="group relative h-full w-full">
-        <div className="absolute m-0 flex h-full w-full flex-col items-center justify-between gap-[10px] rounded-[5px] bg-gradient-to-r from-[#9C9C9C] via-[#D5D5D5] via-5% to-[#f8f8f8] to-20% p-2 shadow-[2px_2px_0px_0px_rgba(0,0,0,0.15)] group-hover:z-10">
+    <div className="size-full bg-cover bg-no-repeat perspective-[1000px]">
+      <div className="group relative size-full">
+        <div className="absolute m-0 flex size-full flex-col items-center justify-between gap-[10px] rounded-[5px] bg-gradient-to-r from-[#9C9C9C] via-[#D5D5D5] to-[#f8f8f8] p-2 shadow-[2px_2px_0px_0px_rgba(0,0,0,0.15)] group-hover:z-10">
           <p className="font-['DVN-Poppins] text-xs leading-5 tracking-wider text-neutral-30 sm:text-sm">
-            {`${abstractPages[1]}...`}
+            {props.highlightAbstract
+              ? (
+                // eslint-disable-next-line react-dom/no-dangerously-set-innerhtml
+                  <div dangerouslySetInnerHTML={{ __html: `${abstractPages[1]}...` }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)